clients/admin-ui/src/pages/management/domains.tsx

```diff
@@ -91,38 +155,41 @@ const CORSConfigurationPage: NextPage = () => {
   };
 
   return (
-    <Layout title="Manage domains">
-      <Box data-testid="cors-configuration">
+    <Layout title="Domains">
+      <Box data-testid="management-domains">
         <Heading marginBottom={4} fontSize="2xl">
-          Manage domains
+          Domains
         </Heading>
         <Box maxWidth="600px">
-          <Text marginBottom={2} fontSize="md">
-            Manage domains for your organization
-          </Text>
           <Text mb={10} fontSize="sm">
```

Comment:
You didn't cause this huge space but let's fix it while we're here
```suggestion
          <Text marginBottom={3} fontSize="sm">
```

Reply:
Good eye 👁️